server/src/main/java/org/opensearch/cluster/ClusterState.java

```diff
@@ -467,6 +473,11 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field("master_node", nodes().getMasterNodeId());
         }
 
+        // Value of the field is identical with the above, and aims to replace the above field.
+        if (metrics.contains(Metric.CLUSTER_MANAGER_NODE)) {
+            builder.field("cluster_manager_node", nodes().getMasterNodeId());
```

Comment:
As noted on a related PR, please create a Github issue to track changing these internal methods to be inclusive

Reply:
In my mind, it's not an internal method, because it's shown in the Javadoc (https://opensearch.org/javadocs/1.3.0/OpenSearch/server/build/docs/javadoc/org/opensearch/cluster/node/DiscoveryNodes.html#getMasterNodeId()).
I think method listed in Javadoc needs an enough period of time to deprecate in advance, before removing.
The issue to track the internal usages is https://github.com/opensearch-project/OpenSearch/issues/1548, and there is a PR out. Maybe I'd better update the description to list usages in different file directories.
(Pasted the related PR here https://github.com/opensearch-project/OpenSearch/pull/2453#discussion_r827498241)